scripts/corfu_server.sh

```diff
@@ -35,9 +35,13 @@ then
     CLASSPATH=`cygpath -wp "$CLASSPATH"`
 fi
 
+if [ "$METRICS_CONFIG_FILE" != "" ]; then
+  LOGBACK_CONFIGURATION="-Dlogback.configurationFile=${METRICS_CONFIG_FILE}"
+  JAVA="$JAVA $LOGBACK_CONFIGURATION"
+fi
```

Comment:
Can you just enable this by default and add the logging information in infrastructure/src/main/resources/logback.prod.xml

Reply:
This was done in order to supply the running processes in the integration tests with an XML file. It's mostly for debugging/viewing metrics in the integration tests. I will enable the metrics in the logback.prod.xml though.